src/coreclr/gc/gcpriv.h

```diff
@@ -5752,7 +5752,7 @@ class heap_segment
     // swept_in_plan_p can be folded into gen_num.
     bool            swept_in_plan_p;
     int             plan_gen_num;
-    int             survived;
+    size_t          survived;
```

Comment:
as I mentioned in the email, we should move this out of this cluster of small types (since they all need to aligned on their natural sizes). so I would move this to be right above gen_num. in this case it happens to be 8-byte aligned already but this is for good practice.